packages/js/product-editor/src/components/iframe-editor/iframe-editor.tsx

```diff
@@ -45,6 +47,7 @@ export function IframeEditor( {
 }: IframeEditorProps ) {
 	const [ resizeObserver, sizes ] = useResizeObserver();
 	const [ blocks, setBlocks ] = useState< BlockInstance[] >( initialBlocks );
+	const [ isInserterOpened, setIsInserterOpened ] = useState( false );
```

Comment:
Noting that in Gutenberg this is retrieved from the `editPostStore` store or `editWidgetsStore`, both of which feel wrong for this use case, though we may consider moving this into the editor settings or context at some point if necessary.